src/main/java/ua/goit/controller/CategoryController.java

```diff
@@ -0,0 +1,25 @@
+package ua.goit.controller;
+
+import ua.goit.dao.CategoryDao;
+import ua.goit.dao.CategoryDaoImpl;
+import ua.goit.model.Category;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+public class CategoryController extends HttpServlet {
+  @Override
+  protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+    resp.setContentType("text/html");
+    CategoryDao categoryDao = new CategoryDaoImpl();
+    PrintWriter writer = resp.getWriter();
+    for (Category category : categoryDao.getAllCategory()) {
+      writer.print("<a href=/kickstarter/Project?categoryName=" + category.getCategoryName() + ">" + category.getCategoryName() + "</a>");
```

Comment:
generated html is a bit quircky. attribute values should be in quotes too so it should be `<a href="link">...` please add a test for that with your next submission